src/Core/Worker.php

```diff
@@ -1140,6 +1149,32 @@ private static function runCron()
 
 		// Cleaning dead processes
 		self::killStaleWorkers();
+
+		// Remove old entries from the workerqueue
+		self::cleanWorkerQueue();
```

Comment:
Please keep the separate `CleanWorkerQueue` class instead of stuffing an already 1,000-lines file, you can call it with `CleanWorkerQueue::execute()`.

Reply:
Each class there is meant to be called as a worker job. It would be confusing to call it directly and not through the worker.

Reply:
Why? The fact what we may not call it through the worker queue doesn't make the feature useless when called directly. In particular, I like that it's in a bite-sized class and not part of a 1,000+ lines wall of code anymore.

Reply:
It's just like calling a `Module` function from somewhere else and not via the router.

Reply:
And I don't see any issue with this either. It's an explicit call so it's easy to trace.

Reply:
From my point of view each class in `src/Worker` or `src/Module` is only called via the worker or the router and does only contain private functions with the exception of the ones with the predefined names like `execute`, `rawContent`, `patch`, `post`, ... 

When a functionality is used not only via this class but also via some other class, then it needs to be placed somewhere else.